lang-go.json5

```
@@ -1,8 +1,8 @@
 {
   "packageRules": [
     {
-      "matchPackagePatterns": [".*giantswarm.*"],
-      "groupName": "giantswarm modules"
+      "matchPackagePatterns": ["^github.com/giantswarm.*"],
+      "groupName": "Giant Swarm Go modules"
```

This is fine as it is. Worth noting that if you want to exclude docker updates and only upgrade go dependencies, you could've also added a `"matchManagers"` like

```suggestion
      "matchPackagePatterns": [".*giantswarm.*"],
      "groupName": "Giant Swarm Go modules",
      matchManagers": ["gomod"]
```

---

I like this better. Thanks!